src/initialization/MOM_shared_initialization.F90

```diff
@@ -501,16 +503,24 @@ subroutine set_rotation_beta_plane(f, G, param_file, US)
     case ("d")
       call get_param(param_file, mdl, "RAD_EARTH", Rad_Earth, &
                    "The radius of the Earth.", units="m", default=6.378e6)
-      y_scl = Rad_Earth/PI
-    case ("k"); y_scl = 1.E3
-    case ("m"); y_scl = 1.
-    case ("c"); y_scl = 1.E-2
+      beta_lat_ref_units = "degrees"
+      y_scl = PI * Rad_Earth/ 180.
+    case ("k")
+      beta_lat_ref_units = "kilometers"
+      y_scl = 1.E3
+    case ("m")
+      beta_lat_ref_units = "meters"
+      y_scl = 1.
     case default ; call MOM_error(FATAL, &
       " set_rotation_beta_plane: unknown AXIS_UNITS = "//trim(axis_units))
   end select
 
+  call get_param(param_file, mdl, "BETA_LAT_REF", beta_lat_ref, &
+                 "The reference latitude (origin) of the beta-plane", &
+                 units=trim(beta_lat_ref_units), default=0.0)
+
   do I=G%IsdB,G%IedB ; do J=G%JsdB,G%JedB
-    f(I,J) = f_0 + beta * ( G%geoLatBu(I,J) * y_scl )
+    f(I,J) = f_0 + beta * ( (G%geoLatBu(I,J) - beta_lat_ref) * y_scl )
```

Comment:
The offset seems wrong to me: f = fo + beta * y has two degrees of freedom for a straight line but adding `beta_lat_ref` here adds up to three parameters.

Reply:
Is it not just a unit conversion?  I don't know if I'd consider that a degree of freedom.

Reply:
Do you mean it would be more correct to redefine `f_0` rather than introduce `beta_lat_ref`?

Reply:
Indeed, `f_0` can be adjusted to get the values you want. I see why `beta_lat_ref` was introduced (namely for convenience) but now there are two ways to get the same result. I guess it's just a formula with parameters, so it's not wrong. I'm not sure if too many parameters is a problem or not.

Reply:
Imho most users expect to input parameters as in: 

https://ocw.mit.edu/courses/earth-atmospheric-and-planetary-sciences/12-808-introduction-to-observational-physical-oceanography-fall-2004/lecture-notes/course_notes_3a.pdf

With f0 being the Coriolis parameter at lat = lat0 (beta_lat_ref).

Reply:
I agree the generality provides the convenience to do it either way and that's probably a good thing.